src/test/java/com/commercetools/sync/customers/CustomerSyncTest.java

```diff
@@ -0,0 +1,322 @@
+package com.commercetools.sync.customers;
+
+import com.commercetools.sync.commons.asserts.statistics.AssertionsForStatistics;
+import com.commercetools.sync.commons.exceptions.ReferenceResolutionException;
+import com.commercetools.sync.commons.exceptions.SyncException;
+import com.commercetools.sync.customers.helpers.CustomerSyncStatistics;
+import com.commercetools.sync.services.CustomerGroupService;
+import com.commercetools.sync.services.CustomerService;
+import com.commercetools.sync.services.TypeService;
+import com.commercetools.sync.services.impl.TypeServiceImpl;
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.customers.Customer;
+import io.sphere.sdk.customers.CustomerDraft;
+import io.sphere.sdk.customers.CustomerDraftBuilder;
+import io.sphere.sdk.models.ResourceIdentifier;
+import io.sphere.sdk.models.SphereException;
+import io.sphere.sdk.types.CustomFieldsDraft;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.CompletionException;
```

Comment:
```suggestion
import java.util.ArrayList;
import java.util.HashSet;
import java.util.List;
import java.util.Optional;
import java.util.concurrent.CompletionException;
import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;
```

Reply:
will not change this, as the import is an automated process by the intellij idea.

Reply:
It depends. Intellij sometimes generates wildcard import for developers. I would rather fix it in that case. Here it is about import ordering. it is just "nice-to-have" and still acceptable if we keep it unchanged.